### ISSUE TITLE
Please Update package_info_plus Version.

### ISSUE THREAD

**Issue by sunpra1** [khalti/checkout-sdk-flutter#15]
Please Update package_info_plus (App is using version ^5.0.1 which depends on web >=0.3.0 <0.5.0). As various popular packages like connectivity_plus, geolocator, firebase_messaging etc. are using web >=0.5.0 <2.0.0 even in versions that are 6-8 months old. Due to this version solving has become impossible.

**Comment by landxcape** [comment]
Yes, package_info_plus: 5.0.1 is 12 months old, and the latest version is 8.1.1. Manually updating the version locally is not an ideal solution.

**Comment by jainmohit2001** [comment]
++

**Comment by jainmohit2001** [comment]
Please update the dependencies for the package.

**Comment by landxcape** [comment]
manually updating the package inside transitive dependencies works so I don't understand why they are still not updating this outdated package

**Comment by jainmohit2001** [comment]
@Biplab-Dutta, please look into this.

**Comment by Biplab-Dutta** [comment]
Hi @jainmohit2001. Thanks for bringing the issue to my attention.
I shall update the versions asap.

**Comment by Biplab-Dutta** [comment]
@jainmohit2001 @landxcape @sunpra1 We apologize for the delayed response to solve this issue. However, I have already filed a PR for the issue mentioned here.
https://github.com/khalti/checkout-sdk-flutter/pull/16
We will have it merged after a thorough testing by our QA.
Thanks again for using our SDK.

**Comment by ravikant-paudel** [comment]
Done

**Comment by ravikant-paudel** [comment]
Done by @Biplab-Dutta 
https://github.com/khalti/checkout-sdk-flutter/pull/16

**Comment by Biplab-Dutta** [comment]
The new version will soon be available on pub. Stay tuned. @landxcape @jainmohit2001 @sunpra1
